tests/phpunit/WC_Stripe_Payment_Gateway_Test.php

```diff
@@ -817,4 +816,59 @@ public function provide_test_payment_icons() {
 			],
 		];
 	}
+
+	/**
+	 * Test that non-retryable error codes return false
+	 *
+	 * @dataProvider non_retryable_error_codes_provider
+	 */
+	public function test_non_retryable_error_codes_return_false( $error_code ) {
+		$error       = new \stdClass();
+		$error->code = $error_code;
+		$error->type = 'invalid_request_error';
+
+		$result = $this->gateway->is_retryable_error( $error );
+
+		$this->assertFalse( $result, "Error code '{$error_code}' should not be retryable" );
+	}
```

Comment:
Not blocking, but the test case for `invalid_request_error` with some other error code is not clearly handled anywhere. `test_retryable_error_types_return_true()` implicitly covers it, but only by having an empty `$error->code` value.